src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -1946,6 +1950,32 @@ abstract class DatabaseTest {
         )
 
     cohortModulesDao.insert(row)
+
+  fun insertVote(
+      projectId: ProjectId,
```

Comment:
A couple things on these insert helper functions:

* ID arguments that tests are likely to pass in can be of type `Any`. Then when they're assigned to properties in the `Row` class, the `.toIdWrapper` extension method turns them into the correct type. The idea is that in tests, we can do, e.g., `insertVote(projectId = 1)` for conciseness.
* For dependencies like this, default to the most recently inserted row of the type in question, which gets tracked   in the `Inserted` class, an instance of which is created as the `inserted` property. Insert functions that create new IDs record the IDs there. The idea is that callers can omit the ID argument entirely if they're setting up some test data.

So this would be `projectId: Any = inserted.projectId` and then in the row creation, `projectId = projectId.toIdWrapper { ProjectId(it) }`.